src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/ProjectHomeViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projecthome
+
+import org.wycliffeassociates.otter.jvm.app.ui.chapterpage.view.ProjectPage
+import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.ProjectCreationWizard
+import tornadofx.*
+
+class ProjectHomeViewModel : ItemViewModel<ProjectHomeModel>(ProjectHomeModel()) {
+
+    val allProjectsProperty = bind(ProjectHomeModel::allProjects)
+
+     fun getAllProjects() = bind(ProjectHomeModel::getAllProjects)
+
+    fun createProject() {
+        find<ProjectCreationWizard> {
```

Comment:
Maybe this logic should go in the model?

Reply:
I ended up have the viewmodel pass the Workspace back to the model so the model could dock/launch the next view. Not sure if that is necessary here